src/composables/navLink.js

```diff
@@ -0,0 +1,36 @@
+import { computed } from 'vue';
+import { useI18n } from 'vue-i18n';
+import { useStore } from 'vuex';
+import {
+  ROUTE_NAME_HOME,
+  ROUTE_NAME_LOGIN,
+  ROUTE_NAME_RULES,
+  ROUTE_NAME_STATS,
+  ROUTE_NAME_LOGOUT
+} from '@/router.js';
+
+export default function usePageLinks() {
```

Comment:
We could use a more explicit rule regarding the use of default vs named exports. I had expected the composable to use a named export because the examples in the Vue docs tend to use named exports, but I don't actually have a specific reason in mind to prefer one over the other for composables, given that they generally export a single function per file.

@haaarsh619 and @seriouslysean which do you think we should use?

Reply:
i am not sure but i think if we are exporting multiple things than we can use named export and for single function than we can use defaults export.

Reply:
@haaarsh619 I'm going to do a deeper dive into the vue docs recommendations but for now let's follow the examples of the docs and use named exports for composables and have them return objects that we can destructure in the consumers